lib/smart_proxy_ipam/phpipam/phpipam_helper.rb

```diff
@@ -1,38 +1,47 @@
 module PhpipamHelper
-  def validate_required_params(required_params, params)
-    err = []
-    required_params.each do |param|
-      if not params[param.to_sym] 
-        err.push errors[param.to_sym]
-      end
-    end
-    err.length == 0 ? [] : {:code => 400, :error => err}.to_json
+  def validate_required_params!(required_params, params)
+    err = required_params.filter { |param| params[param] }.map { |param| errors[param] }
```

Comment:
array.filter is only available since ruby 2.6
array.filter is an alias for array.select